docs/index.md

```diff
@@ -257,6 +257,48 @@ connection "aws_account_z" {
 
 Using named profiles allows Steampipe to work with your existing CLI configurations, including SSO and using role assumption.
 
+### AWS Profile Credentials with IAM Roles
```

Comment:
```suggestion
### AWS Profile Credentials with AssumeRole
```

This slight change should keep section headers consistent (see the other `AssumeRole` sections below)